aws/cloudformation/iam.yml.erb

```diff
@@ -31,6 +31,45 @@ Resources:
     Properties:
       Path: '/'
       Roles: [Ref: FrontendRole]
+  StandaloneFrontendRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: [ec2.amazonaws.com]
+            Action: ['sts:AssumeRole']
+      Path: '/'
+      Policies:
+        - PolicyName: instance
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action: ['s3:GetObject']
+                Resource: 'arn:aws:s3:::*'
+              - Effect: Allow
+                Action:
+                  - 'cloudformation:SignalResource'
+                  - 'cloudformation:DescribeStackResources'
+                  - 'autoscaling:CompleteLifecycleAction'
+                  - 'ec2:DescribeImages'
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - 'dynamodb:*'
+                Resource:
+                  - {'Fn::Join': [':',['arn:aws:dynamodb',{Ref: 'AWS::Region'},{Ref: 'AWS::AccountId'},
+                    table/*_tables
```

Comment:
Yeah, for now this is only used by the `adhoc` environment, so i'll narrow it down further to `adhoc_tables`. (I'll revisit if we ever want to reuse this instance profile for other standalone environments such as `test`/`staging` in the future.)